src/templates/case-study.jsx

```diff
@@ -50,6 +65,18 @@ CaseStudyTemplate.propTypes = {
   shortDescription: PropTypes.string.isRequired,
   tags: PropTypes.arrayOf(PropTypes.string).isRequired,
   success: PropTypes.objectOf(PropTypes.string).isRequired,
+  quotesImage: PropTypes.shape({
```

Comment:
This will now be an object, and you can pull in the PropTypes from the Quotes section component